ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/state/BeaconState.java

```diff
@@ -29,7 +29,7 @@
 
 public class BeaconState {
   // Misc
-  protected long slot;
+  public long slot;
```

Comment:
Why does this need to be public? It is better to use the getter/setters to access and mutate this field

Reply:
Whoops - was testing something out and forgot to change back. Thanks!